aiozipkin/aiohttp_helpers.py

```diff
@@ -36,6 +37,10 @@ def middleware_maker(tracer_key=APP_AIOZIPKIN_KEY,
                 span.tag(HTTP_PATH, request.path)
                 span.tag(HTTP_METHOD, request.method.upper())
 
+                peername = request.remote
+                if peername is not None:
+                    span.remote_endpoint(None, ipv4=peername)
```

Comment:
`peername` could be IPv6 too.
`isinstance(ipaddress.ip_address(peername), ipaddress.IPv4Address)` could help.
In case of `ValueError` the `peername` is UNIX socket path (str) as I mentioned.